checkoutmanager/runner.py

```diff
@@ -82,6 +82,21 @@ def execute_action(dirinfo, custom_actions, action):
         return e
 
 
+def run(action, group=None, custom_actions=None, conf=None, single=False):
+
+    if not conf:
```

Comment:
Do you really need/want to pass `conf` and `custom_actions` here? You can just as easily read them here in `run()` directly without first reading them in `main()` and passing them on. Saves you some parameters.